include/deal.II/grid/reference_cell.h

```diff
@@ -183,6 +192,32 @@ namespace ReferenceCell
     return Point<dim>(+0.0, +0.0, +0.0);
   }
 
+  /**
+   * Create a (coarse) grid with a single cell of the shape of the provided
+   * reference cell.
+   */
+  template <int dim, int spacedim>
+  void
+  create_coarse_triangulation(Triangulation<dim, spacedim> &tria,
+                              const Type &                  reference_cell);
+
+  /**
+   * Return a default linear mapping matching the given reference cell (QGauss,
+   * Simplex::QGauss, Simplex::QGaussPyramid, Simplex::QGaussWedge).
+   */
+  template <int dim, int spacedim>
+  const Mapping<dim, spacedim> &
+  get_default_linear_mapping(const Type &reference_cell);
+
+  /**
+   * Return a default quadrature matching the given reference cell (MappingQ1
+   * for hypercube cells and MappingFE else).
+   */
+  template <int dim>
+  Quadrature<dim>
+  get_default_quadrature(const Type &   reference_cell,
+                         const unsigned n_points_1D);
```

Comment:
Can you document the second argument?

Is there a "default quadrature" for cell types? It's not clear to me why that would be a Gauss quadrature formula. We often use those, but it's not the only correct choice. Maybe `get_gauss_type_quadrature()` would be a better name? Or `get_gauss_quadrature()`?

Reply:
I went with `get_gauss_type_quadrature()`.